### PR TITLE
For building iron/cli Docker image.

### PR DESCRIPTION
**Pull request by treeder**

### CODE REVIEW

File: build-docker.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# This script builds the iron/cli Docker image, tags and pushes it to Docker Hub.
+
+#git clone https://github.com/iron-io/ironcli.git
+#cd ironcli
+
+img=iron/go
+wd=/go/src/github.com/iron-io/ironcli
+docker run --rm -v "$PWD":$wd -w $wd $img go build -o iron
+
+docker build -t iron/cli:latest .
```
Comment (arschles) [inline build-docker.sh]: can we split the build and push steps into different scripts, so it's easier to test locally? maybe leave the above into `build-docker.sh` and put the below in `push-docker.sh`, with `build-docker.sh` first

Comment (treeder) [inline build-docker.sh]: To test locally, you just do the `+docker build -t iron/cli:latest .` step, script isn't really required.

Comment (arschles) [inline build-docker.sh]: to test locally, you'd also have to execute `docker run --rm -v "$PWD":$wd -w $wd $img go build -o iron` before running `docker build`. I think that non-trivial command combined with the simple `docker build` warrants a script

Comment (treeder) [inline build-docker.sh]: True. Too many scripts, some named with docker already, should merge/rename/remove some of them I think.

Comment (arschles) [inline build-docker.sh]: +1. switch over to using your universal tool you blogged about?

File: README.md [hunk 2]
```diff
@@ -15,9 +15,28 @@ Grab the latest version for your system on the [Releases](https://github.com/iro
...
+
+```sh
+export IRON_TOKEN=YOURTOKEN
+export IRON_PROJECT_ID=YOURPROJECT_ID
+```
+
+And then use `-e` flags with the docker run command:
+
+```sh
+docker run --rm -it -e IRON_TOKEN -e IRON_PROJECT_ID -v "$PWD":/app -w /app iron/cli ...
```
Comment (arschles) [inline README.md]: should be `-e $IRON_TOKEN -e $IRON_PROJECT_ID`, no?

**Comment by arschles** [comment]
@treeder made one more comment in docs, assuming that's cleared I'm ok merging this w/o splitting the build script as we discussed (above)

**Comment by treeder** [comment]
@arschles no, it will use the var in the current environment the way it is. ie: the exported ones above it.

**Pull request merged**